Mergin/project_settings_widget.py

```diff
@@ -28,6 +29,16 @@ class ProjectConfigWidget(ProjectConfigUiWidget, QgsOptionsPageWidget):
     def __init__(self, parent=None):
         QgsOptionsPageWidget.__init__(self, parent)
         self.setupUi(self)
+
+        self.cmb_photo_quality.addItem("Original", 0)
+        self.cmb_photo_quality.addItem("High", 1)
+        self.cmb_photo_quality.addItem("Medium", 2)
+        self.cmb_photo_quality.addItem("Low", 3)
```

Comment:
can we get some extra info on what the High/Medium/Low mean? either expected size in MB or expected size in pixels (or both)

Reply:
Added information about approximate size, as I'm not sure about correct wording for pixel size. We have to take into account image orientation so it will be either width or height